packages/stores/src/queries-external/store.ts

```diff
@@ -24,5 +31,13 @@ export class QueriesExternalStore {
       priceStore,
       poolRewardsBaseUrl
     );
+    this.queryIbcDepositStatuses = new ObservableQueryIbcDepositStatuses(
+      kvStore,
+      ibcStatusBaseUrl
```

Comment:
IMO we should combine these into a single store with the same name you originally had. Then (in the same file) you can have a map store for deposits and withdrawals, each mapping from counterpartyChainId->ibcStatusQuery.